code/api/src/modules/user/mutations.js

```diff
@@ -22,6 +22,11 @@ export const userSignup = {
     password: {
       name: 'password',
       type: GraphQLString
+    },
+    // Add style
+    style: {
+      name: 'style',
+      type: GraphQLInt
```

Comment:
We may not need this. We can have a default set when a user signs up to be enum 0 that way FE won't have to pass this as an argument.